bundles.md

```diff
@@ -0,0 +1,131 @@
+# Attestation Bundle
+in-toto attestations support multiple different predicate types with the idea that
+users will want to propagage multiple assertions about a single software package
+(Provenance, Code Review, Test Result, vuln scan, ...).
+
+Attestation Bundles provides a canonical method of storing those attestations in
+a single place.
+
+The attestations stored in a bundle may all come from the same source, signed by
+the same key, or they come from multiple sources, signed by different keys.  New
+attestations would simply be added to the existing bundle.
+
+**NOTE**: Attestation Bundles themselves are **not authenticated** instead each
+individual attestation is authenticated
+([using DSSE](https://github.com/secure-systems-lab/dsse)). As such, an attacker
+might be able to _delete_ an attestation without being detected.  Predicates that
+follow [the monotonic principle](spec/README.md#parsing-rules) should not have any
+issues with this behavior.
+
+## Data structure
+
+Attestation Bundles use [JSON Lines](https://jsonlines.org/) to store multiple
+[DSSEs](https://github.com/secure-systems-lab/dsse).
+
+*  Each line MUST contain a _single_ DSSE.
```

Comment:
Is it OK for a bundle to contain old-style in-toto links and layouts? Since consumers ignore any unrecognized payload types, why not allow other JSON objects as well?

Reply:
I like this idea from a usability perspective, but I'm concerned from a security one.  Since JSON objects don't have standardized typing someone might mis-interpret one object as another.

Reply:
Ah yes, that makes sense.

I think this comment is obsolete since accepting your other suggestion.